src/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(Omega_h_SOURCES
   Omega_h_compare.cpp
   Omega_h_confined.cpp
   Omega_h_conserve.cpp
+  Omega_h_describe.cpp
```

Comment:
I think the other 'utilities' don't have the `Omega_h_` so this could just be `describe.cpp`.